tools/logstash/README.md

```diff
@@ -0,0 +1,54 @@
+# Analyzing Marathon with ELK Stack
+
+Marathon comes with a basic set of grok filter for Logstash to load logs into
+Elasticsearch. This should enable users to analyze logs with Kibana.
+
+## Generating LogStash config
+
+### DCOS Bundle
+
+LogStash config can be generated for a DCOS bundle using the following command:
+
+```
+bin/target.sc <path-to-diagnostic-bundle>
```

Comment:
I do like this approach but find the name `target` confusing. How about `bin/elk generate-config <path-to-diagnostic-bundle>`?

Reply:
Okay if we keep it as is? I personally like the name, and don't find it confusing